src/components/contextpopup/ContextPopupDirective.js

```diff
@@ -114,7 +114,6 @@ goog.require('ga_permalink');
               coord4326['lat'] = coord4326[1];
               scope.coordmgrs = $window.proj4.mgrs.forward(coord4326).
                   replace(/(.{5})/g, '$1 ');
-              scope.altitude = '-';
```

Comment:
Any reason why this was removed? It was put there in case height service fails or is very slow, we still show something.